src/components/PlatformNavigator/InfoPopover.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import { Button, Link, View } from '@aws-amplify/ui-react';
+import { parseLocalStorage } from '@/utils/parseLocalStorage';
+
+export const InfoPopover = ({ platform }) => {
+  const [expanded, setExpanded] = useState(false);
+
+  useEffect(()=>{
+    let returnVisitor = parseLocalStorage('returnVisitor', false);
+    if(!returnVisitor){
+      setExpanded(true);
+      if(typeof localStorage !== 'undefined'){
+        localStorage.setItem('returnVisitor', 'true');
+      }
+    }
+  },[]);
+
+  return (
+    <View className={classNames('info-popover')}>
+      <Link onClick={() => setExpanded(!expanded)} fontWeight={"bold"}>Info</Link>
```

Comment:
For some reason I can't seem to tab to the "Info" link. I think it's because we're creating an `a` tag but it doesn't have an `href`? Could this be a button with the link variation set: https://ui.docs.amplify.aws/react/components/button#variations?